antiplag/tasks.py

```diff
@@ -95,37 +87,37 @@ def compare_documents(
                 # returns percentage representing how similar docs are
                 similarity = text_comparison(doc.text, similar_doc["text"])
 
-                result_similarity += similarity
+                result_similarity += similarity[0][1]
                 compared_count += 1
             except:
                 # TODO: Should uncomparable documents be included?
                 similarity = None
 
-            if similarity > threshold:
+            if similarity[0][1] > threshold:
```

Comment:
If you already added `similarity[0][1]` to `result_similarity` why don't you reuse that variable?

Also, if `similarity` is `None` as in the `except` branch - your code is likely to fail with error.